Formula/hyperkit.rb

```diff
@@ -0,0 +1,120 @@
+class Hyperkit < Formula
+  desc "Lightweight virtualization hypervisor for MacOS"
+  homepage "https://github.com/moby/hyperkit"
+  url "https://github.com/moby/hyperkit.git",
+    :tag => "v0.20170425",
+    :revision => "a9c368bed6003bee11d2cf646ed1dcf3d350ec8c"
```

Comment:
As above, I need to get the commit hash. I'm just following the [boot2docker formula](https://github.com/ilovezfs/homebrew-core/blob/940846866105fbbfa42ebab6fd0321261b000ea4/Formula/boot2docker.rb).